e2e/process-services-cloud/form-field/visibility-condition.e2e.ts

```diff
@@ -248,3 +253,155 @@ describe('Visibility conditions - cloud', () => {
+        await taskFormCloudComponent.formFields().checkWidgetIsHidden('Number2');
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+
+        await taskFormCloudComponent.formFields().setFieldValue(by.id, 'Number1', 5);
+        await taskFormCloudComponent.formFields().checkWidgetIsVisible('Number2');
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+
+        await taskFormCloudComponent.formFields().setFieldValue(by.id, 'Number1', 123);
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        await taskFormCloudComponent.formFields().checkWidgetIsHidden('Number2');
+    });
+
+    it('[C315169] Should be able to start a process with visibility condition for number widgets', async () => {
+        const processModelWithSe = 'numbervisibilityprocess';
+        const numberVisibilityTask = 'number_visibility_task';
+        processName = StringUtil.generateRandomString(5);
+
+        await processCloudDemoPage.openNewProcessForm();
+        await startProcessPage.clearField(startProcessPage.processNameInput);
+        await startProcessPage.selectFromProcessDropdown(processModelWithSe);
+        await startProcessPage.enterProcessName(processName);
+        await startProcessPage.clickStartProcessButton();
+
+        await processDetailsCloudDemoPage.selectProcessTaskByName(processName);
+        await tasksCloudDemoPage.taskListCloudComponent().selectRow(numberVisibilityTask);
+        await taskFormCloudComponent.clickClaimButton();
+
+        await taskFormCloudComponent.formFields().checkWidgetIsVisible('Number1');
+        await taskFormCloudComponent.formFields().checkWidgetIsHidden('Number2');
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+
+        await taskFormCloudComponent.formFields().setFieldValue(by.id, 'Number1', 5);
+        await taskFormCloudComponent.formFields().checkWidgetIsVisible('Number2');
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+
+        await taskFormCloudComponent.formFields().setFieldValue(by.id, 'Number1', 123);
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeTruthy();
+        await taskFormCloudComponent.formFields().checkWidgetIsHidden('Number2');
+
+        await taskFormCloudComponent.formFields().setFieldValue(by.id, 'Number1', 4);
+        await expect(await taskFormCloudComponent.formFields().isCompleteFormButtonDisabled()).toBeFalsy();
+        await taskFormCloudComponent.clickCompleteButton();
+    });
+
+    it('[C315232] Should be able to complete a process with visibility condition for number widgets', async () => {
+        const processModelWithSe = 'booleanvisibilityprocess';
```

Comment:
same as above, makes no sense moving a string to a constant if it's used only once